pkg/security/probe/approvers_basename_bpf.go

```diff
@@ -0,0 +1,53 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
+// +build linux
+
+package probe
+
+import (
+	"path"
+
+	"github.com/DataDog/datadog-agent/pkg/security/rules"
+)
+
+func onNewBasenameApprovers(probe *Probe, eventType EventType, field string, approvers rules.Approvers) ([]activeApprover, error) {
```

Comment:
nit: I think you can remove the `_bpf` to this file ?

Reply:
Done